src/org/usfirst/frc2175/commandmapper/JoystickEventMapper.java

```diff
@@ -27,7 +29,16 @@ public JoystickEventMapper(RobotConfig robotConfig,
         retractCatapult
                 .whenPressed(new RetractCatapultCommand(robotSubsystems));
 
+        JoystickButton runIntakeIn = gamepadConfig.getIntakeInButton();
+        runIntakeIn.whenPressed(
+                new RunRollerbarIntakeInCommand(robotSubsystems, 0));
+
+        JoystickButton runIntakeOut = gamepadConfig.getIntakeOutButton();
+        runIntakeOut.whenPressed(
+                new RunRollerbarIntakeOutCommand(robotSubsystems, 0));
```

Comment:
These don't make sense, because according to how the commands are set up these will run the intake with a speed of 0. But take a look at my overall comment on this PR.